concordia/templates/action-app.html

```diff
@@ -288,7 +289,7 @@ <h5>Keyboard Shortcuts</h5>
             </p>
         </div>
     </main>
-    <footer id="action-app-footer" class="footer fixed-bottom bg-light">
+    <footer id="action-app-footer" class="footer fixed-bottom bg-light" style="display: none;">
```

Comment:
The `style` attribute feels like something which should be in the associated CSS in general, but in this specific case I'm wondering whether we should just delete the footer since support is being removed for it in other places.

Reply:
Let's get rid of it.

Reply:
Comment out lines 538 and 348 of static/js/action-app/index.js and then the asset list will load without the footer.

Reply:
Yeah, that was it – the browser errors were triggered by the code which updated the counter in the footer. This commit removes those along with the footer: https://github.com/LibraryOfCongress/concordia/commit/b5ab82e73bc5cfcf4b862a45e82d4a827dc13ead

Reply:
After you've done a Git fetch, you can `git cherry-pick b5ab82e73bc5cfcf4b862a45e82d4a827dc13ead` to pull that onto your branch.

Reply:
I don't see any errors. @rstorey lines 348 and 538 in `static/js/action-app/index.js` are blank for me. Can you give me little more info on this?

Reply:
> I don't see any errors. @rstorey lines 348 and 538 in `static/js/action-app/index.js` are blank for me. Can you give me little more info on this?

If you cherry-picked Chris' commit, then you don't need to do this.

Reply:
Are there any errors in the console?